config/environments/production.rb

```diff
@@ -83,4 +83,7 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.hosts << "barcelona.degica.com"
+  
 end
```

Comment:
I think you have to exclude a path for ALB health check.
Similar issue is [here](https://github.com/degica/hcs/pull/501).

Reply:
Thank you for that! I was worried I was missing something else. Updated to include the fix.